### PR TITLE
TypeScript Declaration fixes

### PR DESCRIPTION
**Pull request by jpattersonz**
Fix typescript declarations for enum exports, Queryable<T> count, and duplicate module name exports

### CODE REVIEW

File: src/dts.ts [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
...n;\n    type GeographyMultiPoint = $data.GeographyMultiPoint;\n    type GeographyMultiPolygon = $data.GeographyMultiPolygon;\n    type GeographyMultiLineString = $data.GeographyMultiLineString;\n    type GeographyCollection = $data.GeographyCollection;\n    type GeometryPoint = $data.Geometry;\n    type GeometryLineString = $data.GeometryLineString;\n    type GeometryPolygon = $data.GeometryPolygon;\n    type GeometryMultiPoint = $data.GeometryMultiPoint;\n    type GeometryMultiPolygon = $data.GeometryMultiPolygon;\n    type GeometryMultiLineString = $data.GeometryMultiLineString;\n    type GeometryCollection = $data.GeometryCollection;\n    type Primitive =\n        Boolean | Binary | Guid | DateTime | DateTimeOffset | Duration | TimeOfDay | Date | Time |\n        Decimal | Single | Float | Double | Int16 | Int32 | Int64 | Byte | SByte | String |\n        GeographyPoint | GeographyLineString | GeographyPolygon | GeographyMultiPoint | GeographyMultiLineString | GeographyMultiPolygon | GeographyCollection |\n        GeometryPoint | GeometryLineString | GeometryPolygon | GeometryMultiPoint | GeometryMultiLineString | GeometryMultiPolygon | GeometryCollection;\n}\nexport {Edm as Edm}"
```
Comment (jpattersonz) [inline src/dts.ts]: added Queryable.count() in here, which is hard to spot via the GitHub diff interface.

**Comment by jpattersonz** [comment]
resolves #5

**Comment by jpattersonz** [comment]
Any feedback?  I'd like to get this merged so I don't have to patch my .d.ts after every run of jaysvcutil.

**Comment by lazarv** [comment]
Hi @jpattersonz,
your contribution is welcome! Thanks for the fix and the pull request, it's now merged into master and the new version is published on npm.

**Pull request merged**